lib/devices/ios/ios-controller.js

```diff
@@ -22,8 +22,11 @@ var uuid = require('uuid-js')
   , IOSPerfLog = require('./ios-perf-log')
   , errors = require('../../server/errors.js')
   , isWindows = require('appium-support').system.isWindows
+  , localIp = require('appium-support').util.localIp
   , NotImplementedError = errors.NotImplementedError
-  , NotYetImplementedError = errors.NotYetImplementedError;
+  , NotYetImplementedError = errors.NotYetImplementedError
+  , Parser = require('../../server/parser.js')
+  , Url = require('url');
```

Comment:
well, we use its Class methods. I'ts kinda funky, we're not calling `new Url()` anywhere, but we are using it as a urlObject factory and then calling class functions like `Url.resolve()` and such, just felt right. But if you think that's inconsistent, no problem!